lib/lynis/lynis_run.pm

```diff
@@ -0,0 +1,92 @@
+# Copyright (C) 2021 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Integrate the Lynis scanner into OpenQA: run test cases/modules
+#          according to section list
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#78224: poo#78230
```

Comment:
1. Is there testcase (tc# ) can be recorded here?
2. It looks like you should put " , " intead of " : " , right?

Reply:
1. Good question: There are no regression test cases here, developer only want to use it to check "maintenance release process". But Security QA will do openQA review as usual.
2.  poo#78224 is the parent poo so I used ":" not ",". do you think it is not proper?

Reply:
1. Got it.
2. I think it is just the symbol misunderstanding to me, I just think if the PR are combined 4 poo, we can list them in the same time. such as:

(But it depends on you :-)

parent poo   :

- poo#78224

 sub poo: 

1.  poo#78330 
2.  poo#88155 
3. [**poo#78230**]

Reply:
I will revise.